src/components/Header/index.tsx

```diff
@@ -97,6 +100,25 @@ export const Header: React.FC<HeaderProps> = ({
 
     return items;
   };
+  const getDocsButton = () => {
+    const items = [
+      <div
+        key="documentation-button"
+        aria-label="documentation-button"
+      >
+        <DocumentationButton
+          key="documentation-button"
+          type="link"
+        >
+        </DocumentationButton>
+
+      </div>
```

Comment:
Why do we need a wrapping `<div>`?

Reply:
other components used a wrapping `<div>` hence I chose to also wrap it in a div.